working/manual_checks_after_generic.sql

```diff
@@ -726,6 +732,148 @@ ORDER BY flag,
     concept_code
 ;
 
+-- 02.13.02 Check for non-Procedure vocabs
```

Comment:
And where is 02.13.01?
@m-khitrun If we have one description for both of them, it should be structurally clear